src/content/operations/usage/_index.en.md

```diff
@@ -684,6 +684,17 @@ are ignored.
 always returns the ClusterIP Service on **cluster2**. Similarly, `cluster3.nginx.nginx-test.svc.clusterset.local` always returns the
 ClusterIP Service on **cluster3**.
 
+##### Cluster Set Virtual IP
+
+Submariner can also allocate a cluster set virtual IP for an exported service that is stored in the `ServiceImport` resource. This is
+an opt-in feature that can be enabled per service via the `lighthouse.submariner.io/use-clusterset-ip` annotation on the `ServiceExport`
+or automatically for all services via the `enable-clusterset-ip` option on `subctl deploy-broker`. Submariner will allocate a virtual IP
+from a pool of IP addresses based on a configurable CIDR assigned to the cluster from a global CIDR range. The first cluster to export
+a service will allocate and assign the virtual IP.
+
+Lighthouse DNS will return the cluster set virtual IP from queries instead of a constituent cluster IP address. However, Submariner
+does not perform load balancing and relies on some external component to do so.
```

Comment:
This gives the impression that the virtual IP won’t work unless the user does something else — is that so?

Reply:
How is the routing handled for the virtualIP? Are we adding any iptable rules to support the datapath?

> Lighthouse DNS will return the cluster set virtual IP from queries instead of a constituent cluster IP address. However, Submariner does not perform load balancing and relies on some external component to do so.

Normally when we return the clusterIP itself (or the associated GlobalIngressIP) kube-proxy does the load-balancing once the remote traffic lands in the local cluster (where the service is exprted). Can you please eloborate how this use-case works with virtualIP?

Reply:
It seems to me that this should be documented because users could end up shooting themselves in the foot.

Can we make the caveat more general? It’s not just load balancing, it’s any kind of routing.

```suggestion
Lighthouse DNS will return the cluster set virtual IP from queries instead of a constituent cluster IP address. However, Submariner
does not perform routing and relies on some external component to do so.
```

Reply:
>Currently we're not doing any routing. This feature currently will be used only by Istio and it will take care of routing. Current submariner connectivity doesn't support this use case.

Can you point me to any links/doc on how this will be used in Istio?

Reply:
> Can you point me to any links/doc on how this will be used in Istio?

The use case was presented on an App Eng Exchange call a few weeks ago. The only doc I know of is https://docs.google.com/document/d/1RXjvYTKMKDFM8KNI9X3HT5M6oqKKwkMde7GeuKLD93Q/edit#heading=h.7mxk48pcmgyk shared by Jacek Ewertowski.

Reply:
0.19.0 has been released so revisiting this PR.  @sridhargaddam Have you been able to review the Istio use case?